### PR TITLE
fix: apply_network_config: false for 18.04-LTS

### PR DESCRIPTION
**Pull request by jackfrancis**
**Reason for Change**:

Apply `apply_network_config: false` cloud-init configuration to 18.04-LTS vms always, not just in a VHD CI scenario.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/cloud-init/artifacts/cse_main.sh [hunk 1]
```diff
@@ -220,7 +223,11 @@ if [[ -n "${MASTER_NODE}" ]]; then
     fi
     time_metric "WriteKubeConfig" writeKubeConfig
     if [[ -z "${COSMOS_URI}" ]]; then
-        time_metric "EnsureEtcd" ensureEtcd
+        if [ "$FULL_INSTALL_REQUIRED" = "true" ]; then
```
Comment (CecileRobertMichon) [inline parts/k8s/cloud-init/artifacts/cse_main.sh]: why is this change needed? It breaks the assumption that you can override the etcd version in the VHD by specifying a custom version.

Comment (jackfrancis) [reply parts/k8s/cloud-init/artifacts/cse_main.sh]: `ensureEtcd` depends upon eth0 having the correct IP address. In 18.04-LTS + non-VHD eth0 does not have the correct address.

Comment (CecileRobertMichon) [reply parts/k8s/cloud-init/artifacts/cse_main.sh]: what about the VHD scenario? with this change we would be skipping ensureEtcd if using VHD

Comment (jackfrancis) [reply parts/k8s/cloud-init/artifacts/cse_main.sh]: Yeah that's a bug will fix

File: parts/k8s/cloud-init/artifacts/label-nodes.service [hunk 2]
```diff
@@ -5,4 +5,6 @@ After=kubelet.service
 Restart=always
 RestartSec=60
 ExecStart=/bin/bash /opt/azure/containers/label-nodes.sh
+[Install]
```
Comment (jackfrancis) [inline parts/k8s/cloud-init/artifacts/label-nodes.service]: @mboersma FYI, these missing lines were preventing `systemctl enable` from working (and a systemd bug that returned 0 upon failure was hiding this from us)

File: examples/kubernetes-ubuntu-distros.json [hunk 3]
```diff
@@ -16,6 +16,12 @@
         "count": 2,
         "vmSize": "Standard_D2_v3",
         "distro": "ubuntu"
+      },
```
Comment (jackfrancis) [inline examples/kubernetes-ubuntu-distros.json]: With these changes we should be able to E2E test ubuntu 18.04-LTS non-VHD

File: parts/k8s/cloud-init/artifacts/cse_main.sh (hunk 1 above)
Comment (CecileRobertMichon) [reply parts/k8s/cloud-init/artifacts/cse_main.sh]: @jackfrancis this wasn't fixed before the PR merged
